Mathlib/MeasureTheory/Function/ConvergenceInMeasure.lean

```diff
@@ -226,12 +284,125 @@ theorem TendstoInMeasure.exists_seq_tendstoInMeasure_atTop {u : Filter ι} [NeBo
 theorem TendstoInMeasure.exists_seq_tendsto_ae' {u : Filter ι} [NeBot u] [IsCountablyGenerated u]
     {f : ι → α → E} {g : α → E} (hfg : TendstoInMeasure μ f u g) :
     ∃ ns : ℕ → ι, ∀ᵐ x ∂μ, Tendsto (fun i => f (ns i) x) atTop (𝓝 (g x)) := by
```

Comment:
```suggestion
    ∃ (ns : ℕ → ι) (_ : Tendsto ns atTop u), ∀ᵐ x ∂μ, Tendsto (fun i => f (ns i) x) atTop (𝓝 (g x)) := by
```
This should be free now

Reply:
Yes, you are right, this now also works. Added...